spyder/api/plugin_registration/confpage.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Spyder completion plugin configuration page."""
+
+# Third party imports
+from qtpy.QtWidgets import (QGroupBox, QVBoxLayout, QCheckBox,
+                            QGridLayout, QLabel)
+
+# Local imports
+from spyder.config.base import _
+from spyder.config.manager import CONF
+from spyder.api.preferences import PluginConfigPage
+from spyder.api.plugins import SpyderPlugin
```

Comment:
```suggestion
from spyder.api.plugins import SpyderPlugin
from spyder.api.preferences import PluginConfigPage
from spyder.config.base import _
from spyder.config.manager import CONF
```

Organize import in alphabetical order.